programs/ProgressBarDemo.hs

```diff
@@ -48,13 +48,65 @@ drawUI p = [ui]
       zBar = overrideAttr P.progressCompleteAttr zDoneAttr $
              overrideAttr P.progressIncompleteAttr zToDoAttr $
              bar $ _z p
-      lbl c = Just $ show $ fromEnum $ c * 100
+      -- custom bars
+      cBar1 = overrideAttr P.progressCompleteAttr cDoneAttr1 $
```

Comment:
Thanks for adding some examples to the demo program. I think only three or four at most would be enough to demonstrate the feature. Can you trim this down to just a few custom bars, say, 4?

Reply:
Done. Added 3 custom examples only.

![3examples](https://github.com/user-attachments/assets/d1cb1345-16d4-4976-b0eb-737ee1a1c48e)